mne/digitization/_utils.py

```diff
@@ -0,0 +1,393 @@
+# -*- coding: utf-8 -*-
+# Authors: Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#          Matti Hamalainen <msh@nmr.mgh.harvard.edu>
+#          Teon Brooks <teon.brooks@gmail.com>
+#          Stefan Appelhoff <stefan.appelhoff@mailbox.org>
+#          Joan Massich <mailsik@gmail.com>
+#
+# License: BSD (3-clause)
+
+import datetime
+import os.path as op
+import re
+
+import numpy as np
+
+from ..utils import logger
+from ..utils import warn
+
+from ..io.constants import FIFF
+from ..io.tree import dir_tree_find
+from ..io.tag import read_tag
+from ..io.write import start_file
+from ..io.write import end_file
+from ..io.write import write_dig_points
+
+from ..transforms import apply_trans
+from ..transforms import als_ras_trans
+from ..transforms import get_ras_to_neuromag_trans
+from ..transforms import Transform
+from ..transforms import combine_transforms
+from ..transforms import invert_transform
+from ..transforms import _to_const
```

Comment:
You prefer one `import` at a time?

Reply:
we don't have a strict policy that I'm aware of, and its easier like this when I'm moving code around. I'll do a clean up on the last PR.